AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/AccessibilityInsightsForAndroidService.java

```diff
@@ -140,6 +143,16 @@ public void onAccessibilityEvent(AccessibilityEvent event) {
     }
   }
 
+  @Override
+  public void onConfigurationChanged(Configuration newConfig) {
+    super.onConfigurationChanged(newConfig);
+    if(newConfig.orientation == this.deviceOrientationHandler.getOrientation()){
```

Comment:
Feels like this is already being handled inside the orientation handler already so this shouldn't be necessary here.

(I'd prefer to be checking in the orientation handler anyways)

Reply:
This is a good call, I've made this change.